packages/teams-js/src/public/geoLocation.ts

```diff
@@ -46,7 +50,14 @@ export namespace geoLocation {
     if (!isSupported()) {
       throw errorNotSupportedOnPlatform;
     }
-    return sendAndHandleSdkError('location.getLocation', { allowChooseLocation: false, showMap: false });
+    return sendAndHandleSdkErrorWithVersion(
+      getApiVersionTag(ApiVersionNumber.V_2, ApiName.GeoLocation_GetCurrentLocation),
```

Comment:
Should this file also be one of the "every function in this file is a V_2 function" comment files?